frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -187,7 +213,11 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
         ? [dataConnection.existing]
         : [];
 
-    const pvcDetails = await createPvcDataForNotebook(projectName, storageData).catch(handleError);
+    const pvcDetails = await createPvcDataForNotebook(
+      projectName,
+      storageData,
+      storageClassNameForPVCAssembly,
```

Comment:
...if you follow my last comment.

```suggestion
      storageClass?.metadata.name,
```